content/zh/docs/hertz/tutorials/basic-feature/protocol/http2.md

```diff
@@ -0,0 +1,2520 @@
+---
+title: "HTTP2"
+date:  2022-12-12
+weight: 2
+description: >
+
+---
+HTTP/2是对HTTP "在线 "表达方式的一种替代。它并不是对协议的彻底重写；HTTP方法、状态码和语义都是一样的，而且应该可以使用与HTTP/1.x相同的API（可能会有一些小的补充）来表示协议。
+
+该协议的重点是性能；具体而言，最终用户感知的延迟、网络和服务器资源的使用。一个主要目标是允许使用从浏览器到网站的单一连接。
+
+Hertz 同时支持 h2 和 h2c。参考了 [net/http2](https://github.com/golang/net/tree/master/http2) 的实现。
+
+
+
+## 示例代码
+
+### h2
+
+```go
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/cloudwego/hertz/pkg/app"
+	"github.com/cloudwego/hertz/pkg/app/client"
+	"github.com/cloudwego/hertz/pkg/app/server"
+	"github.com/cloudwego/hertz/pkg/network/standard"
+	"github.com/cloudwego/hertz/pkg/protocol"
+	"github.com/hertz-contrib/http2/config"
+	"github.com/hertz-contrib/http2/factory"
+)
+
+const (
+	keyPEM = `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
The specific key content can use `keyPEM = "your key PEM"` instead. There is no need to show a complete keyPEM